app/views/staff/activity_uploads/new.html.haml

```diff
@@ -8,7 +8,17 @@
 
   .govuk-grid-row
     .govuk-grid-column-two-thirds
-      %p.govuk-body.govuk-heading-m
+      %h2.govuk-heading-m
         = link_to t("action.activity.download.button"), report_activity_upload_path(@report_presenter, format: :csv), class: "govuk-link"
 
-  = render partial: "upload_form"
```

Comment:
Can this partial be deleted now?

Reply:
I'd rather not, because it's still being used on the post-upload page, update.html.haml. The label and hint are different, because update serves as the recover-from-error page.

Reply:
Ah, fair enough. I missed it was being used elsewhere. Merge away!